docroot/themes/humsci/humsci_basic/config/install/humsci_basic.settings.yml

```diff
@@ -2,4 +2,3 @@ brand_bar_variant_classname: default
 lockup:
   option: default
 global_footer_variant_classname: default
-animation_enhancement: false
```

Comment:
hmm but I think we still need this in default config - I need to ask Kasey tomorrow. https://github.com/SU-HSDO/suhumsci/blob/develop/config/default/humsci_colorful.settings.yml

Reply:
@heatherdesigns you can keep this change. it doesn't hurt, but I also don't know that these files do much anymore